src/main/java/mekanism/common/integration/MekanismHooks.java

```diff
@@ -393,4 +402,38 @@ private void addMetallurgy() {
         OreDictManager.addStandardOredictMetal("Vyroxeres");
         OreDictManager.addStandardOredictMetal("Zinc");
     }
+
+    private void registerMAEnrichmentRecipe(int itemMeta, int oreType) {
```

Comment:
Not a fan of using magic numbers like this, and the double ternary is quite bad.

Is there absolutely no way to get this info from an api or anything? If not a couple of enums would do nicely that have prefixes as a member field & the input/output meta on another enum's fields.

Same goes for the Combiner ones - methods look like they could be combined anyway?